src/app/storage/defaultDashboard.ts

```diff
@@ -69,5 +69,15 @@ export const defaultDashboard: DashboardLayout = {
         height: 330,
       },
     },
+    {
+      id: 'fastercarsfrombehind',
+      enabled: true,
```

Comment:
Should we set this to off for now until we add the settings page for it?
```suggestion
      enabled: false,
```